src/util.js

```diff
@@ -21,28 +21,26 @@ export function encodeEntities(str) {
 	// Seek forward in str until the next entity char:
 	for (; i < str.length; i++) {
 		switch (str.charCodeAt(i)) {
-			case 60:
-				ch = '&lt;';
-				break;
-			case 62:
```

Comment:
It isn't needed - there's no way to close a tag from within a quoted attribute.

Reply:
I'm worried about this change as we use the same encoding function for `Text` too.

Reply:
Same deal for text - since there's no way to open a tag, there's no need to block the closing of a tag. 

I was a little unsure too, but I can't think of any cases where this could lead to actual HTML parsing. It's also been removed from Svelte and Inferno.